src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
```

요거 없어도 되지 않나요?! (다른 모든 파일 포함)

Reply:
에러가 나서 지속적으로 Import 해줬었는데, eslint 의 `react/react-in-jsx-scope` 속성 때문에 그랬었군요 😅. 불필요한 `React` import 문 제거했습니다!